src/Planning/understand-options-tfs-vsts-environments.md

```diff
@@ -45,7 +43,7 @@ For TFS there are a few deployment options.
 
 ## Capacity Planning
 
-To decide which of the TFS deployment options are viable, you need to consider the advantages of each option, the number of users and projects you will need to support and understand the expected requests per second (RPS) per application tier (AT) server. 
+To decide which of the Azure DevOps Server deployment options are viable, you need to consider the advantages of each option, the number of users and projects you will need to support and understand the expected requests per second (RPS) per application tier (AT) server. 
```

Comment:
We will need to rewrite this since the RPS metric is no longer being generated. 
```suggestion
To decide which of the Azure DevOps Server deployment options are viable, you need to consider the advantages of each option and the number of users and projects you will need to support. 
```